sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -141,7 +145,8 @@ class PlanResolutionSuite extends AnalysisTest {
       CTESubstitution,
       new ResolveCatalogs(catalogManager),
       new ResolveSessionCatalog(catalogManager, conf, _ == Seq("v")),
-      analyzer.ResolveTables)
+      analyzer.ResolveTables,
```

Comment:
I wasn't quite sure whether this test is only for `ResolveTables`. Any thoughts @cloud-fan?

Reply:
We should just test `ResolveRelations`, which calls `ResolveTables`